services/movie.py

```diff
@@ -1,14 +1,19 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
+    title: str = None,
```

Comment:
make type hints accurate everywhere. If it can be None by default, it means you need to add | None